Sources/MapboxNavigation/CarPlayManagerDelegate.swift

```diff
@@ -324,6 +324,51 @@ public protocol CarPlayManagerDelegate: AnyObject, UnimplementedLogging {
                         routeCasingLineLayerWithIdentifier identifier: String,
                         sourceIdentifier: String,
                         for parentViewController: UIViewController) -> LineLayer?
+    
+    // MARK: Notifications Management
+    
+    /**
+     Determines if the maneuver should be presented as a notification when the app is in the
+     background.
```

Comment:
Curious why it can't control notifications when the app isn't in the background?

Reply:
It seems that CarPlay behavior is similar to iOS, when iOS app is in foreground notification will not be shown. Interestingly `UNUserNotificationCenter` also allows to show generic notification on CarPlay using something like:
```swift
UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .sound, .badge, .carPlay]) { granted, error in
    UNUserNotificationCenter.current().getNotificationSettings { notificationSettings in
        print("carPlaySetting: \(notificationSettings.carPlaySetting.rawValue)")
    }
}
```

but `carPlaySetting` is always disabled. Not sure if it's a problem on my side or it's not really supported after all.